assets/php/class-wgpb-block-grid-base.php

```diff
@@ -205,7 +204,7 @@ public function render() {
 
 		$output = implode( '', array_map( array( $this, 'render_product' ), $products ) );
 
-		return sprintf( '<ul class="%s">%s</ul>', esc_attr( $classes ), $output );
+		return sprintf( '<div class="woocommerce"><ul class="%s">%s</ul></div>', esc_attr( $classes ), $output );
```

Comment:
The container classes should be on the container element, so I think this should get updated so that `get_container_classes` returns `woocommerce` too, and then this would be:

```suggestion
		return sprintf( '<div class="%s"><ul class="products">%s</ul></div>', esc_attr( $classes ), $output );
```